kotlinx-coroutines-core/common/src/CancellableContinuation.kt

```diff
@@ -168,36 +173,59 @@ public interface CancellableContinuation<in T> : Continuation<T> {
     @ExperimentalCoroutinesApi
     public fun CoroutineDispatcher.resumeUndispatchedWithException(exception: Throwable)
 
+    /** @suppress */
+    @Deprecated(
+        "Use the overload that also accepts the `value` and the coroutine context in lambda",
+        level = DeprecationLevel.WARNING
+    ) // warning since 1.9.0, was experimental
+    public fun resume(value: T, onCancellation: ((cause: Throwable) -> Unit)?)
+
     /**
-     * Resumes this continuation with the specified `value` and calls the specified `onCancellation`
-     * handler when either resumed too late (when continuation was already cancelled) or, although resumed
-     * successfully (before cancellation), the coroutine's job was cancelled before it had a
-     * chance to run in its dispatcher, so that the suspended function threw an exception
-     * instead of returning this value.
+     * Resumes this continuation with the specified [value], calling the specified [onCancellation] if and only if
+     * the [value] was not successfully used to resume the continuation.
+     *
+     * The [value] can be rejected in two cases (in both of which [onCancellation] will be called):
+     * - Cancellation happened before the handler was resumed;
+     * - The continuation was resumed successfully (before cancellation), but the coroutine's job was cancelled before
+     *   it had a chance to run in its dispatcher, and so the suspended function threw an exception instead of returning
+     *   this value.
      *
      * The installed [onCancellation] handler should not throw any exceptions.
-     * If it does, they will get caught, wrapped into a [CompletionHandlerException] and
+     * If it does, they will get caught, wrapped into a [CompletionHandlerException], and
```

Comment:
This is effectively an internal class with no documentation. Probably backticked version will do

Reply:
Yes, it's creating a link to 404: https://kotlinlang.org/api/kotlinx.coroutines/kotlinx-coroutines-core/kotlinx.coroutines/-cancellable-continuation/invoke-on-cancellation.html The reason is https://github.com/Kotlin/dokka/issues/3601, but there is no reason not to work around it here.